pkg/config/config.go

```diff
@@ -87,13 +91,26 @@ func ComposeConfig(infrastructure *configv1.Infrastructure, clusterProxy *config
 		return OperatorConfig{}, err
 	}
 
+	featureGatesString := ""
+	upstreamGates, err := util.GetUpstreamCloudFeatureGates()
+	if err != nil {
+		klog.Errorf("Unable to get upstream feature gates: %s", err)
+		return OperatorConfig{}, err
```

Comment:
Please wrap the error adding additional context before passing back

Reply:
In this context we don't wrap errors, look at L80+ of the same file. Do you still want to wrap it here (and then in all the 3 instances of returning `err` in this function, or nowhere?

Reply:
We should be wrapping the errors everywhere, it's a legacy mistake that we aren't in some places